### PR TITLE
Add post for video

### PR DESCRIPTION
**Pull request by theScrabi**
add post for the video

### CODE REVIEW

**Comment by TheAssassin** [comment]
PR is invalid as it contains the history of #44 as well. @theScrabi please remove the offending commit.

**Comment by TheAssassin** [comment]
@theScrabi should we embed the video player in the post?

**Comment by theScrabi** [comment]
Yea why not.

File: _data/images.yml [hunk 1]
```diff
@@ -7,3 +7,9 @@ new:
     name: 34C3 logo
     author: Gregor Sedlag
     origin: https://engelsystem.de/img/34c3_logo.png
+
+34c3-press:
+    url: 34c3-press.png
+    license:
```
Comment (TobiGr) [inline _data/images.yml]: Please add at least your name to this, so people know where we got this from. The new entry also needs an author.

File: _posts/2018-02-12-NP-talk.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+author: Schabi
+image: 34c3-press
+---
+
+As you may know form the previous post, I did a little talk about NewPipe at the last year CCC congress in Leipzig.
+This time we managed to take a [video](https://media.assassinate-you.net/video/newpipe-on-34c3) of it.
+
+I wanted to give a little impression about what happened last year, and what is planed for the future.
+
+[![The Video](/img/talkshot.jpg)](https://media.assassinate-you.net/video/newpipe-on-34c3)
```
Comment (TobiGr) [inline _posts/2018-02-12-NP-talk.md]: We should think of a good style for images in posts.

**Comment by TobiGr** [comment]
>  should we embed the video player in the post?

Good idea. I'll do it.

**Comment by TheAssassin** [comment]
@TobiGr I am already working on it. The software I use for hosting the video at the moment doesn't support embedding, though (a lot of similar functionality, like quality selection etc., are available as paid plugins...). I'll push something as soon as it works reliably.

You could help me, though. How can I force a "Read more" link inside the text? I would like to embed the video only in the article's page, not the overview.

**Comment by TobiGr** [comment]
@TheAssassin Oh. I didn't read your text. Sorry. That was too fast.
YouPHPTube supports embedding videos. Click "share" and embedding is one option.

> You could help me, though. How can I force a "Read more" link inside the text? I would like to embed the video only in the article's page, not the overview.

Good question. We can prevent loading the iframe by default and then enable it on post pages. But what do you want to do with the "read more" link? There is already one.

**Comment by TheAssassin** [comment]
@TobiGr as said before, the idea is to show the embedded video only on the article page, not in the article overview.

**Comment by TheAssassin** [comment]
> YouPHPTube supports embedding videos. Click "share" and embedding is one option.

I didn't see a "Share" button because I was logged in, I found the option now.

**Comment by TobiGr** [comment]
Because we have an iframe we can't prevent it loading by default like we can do it with videos. We could enable/disable the src attribute by renaming or adding it. This is more a hack than good development but I can't think of an other solution right now. How do you feel about it?

**Comment by TheAssassin** [comment]
@TobiGr what I want to do is forcing the page to be split into "shown on overview page" -> "Read more" -> "rest of page" by inserting this "Read more" link with some macro, as you can do with literally *any* other blog software, would solve the problem, as the `<iframe>` tag wouldn't even show up any more.

As this might be a desired feature to split up the page manually, e.g., for a longer blog post with a proper abstract before the main content, I'd like to know how this could be achieved. What plugin are we using for this "Read more" stuff?

**Comment by TobiGr** [comment]
Now I get what you want to do :) We don't use a plugin for that. We cut the whole thing after 100 words, which isn't a problem as long as we only use texts and no HTMl.

**Comment by TobiGr** [comment]
[This](https://jekyllrb.com/docs/posts/#post-excerpts) looks good. It requires a restructure, but that worth it.

**Comment by TheAssassin** [comment]
Thanks for implementing this, @TobiGr. I was already looking for the plugin in use, but couldn't find it, so I figured I should ask you.

**Comment by TobiGr** [comment]
Should be good to merge now.

**Comment by TheAssassin** [comment]
Deployed the new version.

**Pull request merged**